lib/src/soloud.dart

```diff
@@ -1017,17 +1037,11 @@ interface class SoLoud {
   /// to `1.0` (meaning full volume).
   ///
   /// Throws [SoLoudNotInitializedException] if the engine is not initialized.
-  ///
-  /// Note that if you `setGlobalVolume()` to `0.8` and then
-  /// `getGlobalVolume()`, you might get a slightly different number,
-  /// such as `0.800000042353`.
-  /// This is expected since the internal audio engine uses float
-  /// instead of double, and so there are rounding errors.
```

Comment:
Why remove this?

Reply:
I still have that note! Don't know, but on GitHub I see this
![aaa](https://github.com/alnitak/flutter_soloud/assets/192827/b6658182-f8a2-4065-a017-6b9224c1f26a)
I have that for `[s|g]etGlobalVolume()` and `[s|g]volume()`. Maybe you don't have pulled something?

Reply:
Oh, interesting. Maybe some GitHub bug. I was seeing this in the GitHub review UI. I'll have another look but I'll assume it's fine for now.